tests/security/atsec/setup_atsec_env.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Download the test scripts which Atsec tests need
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#108485
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install tool packages
+    zypper_call('in wget');
+
+    # Download the test scripts
+    my $code_path = get_required_var('CODE_PATH');
```

Comment:
@Amrysliu Can we put the scripts into ~data dir?

Reply:
The code/data put in` "data/"` is opensource this `*.c` file is from ATSEC and is confidential. @rfan1